src/ptvsd/_vendored/pydevd/_pydev_runfiles/pydev_runfiles_pytest2.py

```diff
@@ -68,7 +69,9 @@ def start_redirect():
 
 
 def get_curr_output():
-    return State.buf_out.getvalue(), State.buf_err.getvalue()
+    buf_out = State.buf_out
+    buf_err = State.buf_err
```

Comment:
Note: this is just a nitpick I ended up including in the pull request, not really relevant (just synching with `pydevd`).